crypto/secp256r1/codec.go

```diff
@@ -0,0 +1,27 @@
+package secp256r1
+
+import (
+	"github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+const (
+	PrivKeyAminoName = "cosmos-sdk/PrivKeySr25519"
+	PubKeyAminoName  = "cosmos-sdk/PubKeySr25519"
+
+	// SignatureSize is the size of an Edwards25519 signature. Namely the size of a compressed
+	// Sr25519 point, and a field element. Both of which are 32 bytes.
+	SignatureSize = 64
```

Comment:
Please adjust this for secp256r1 and remove the stuff about Sr25519 and Edwards25519.